core/runtime/src/test/java/io/quarkus/runtime/configuration/ConfigExpanderTestCase.java

```diff
@@ -24,31 +23,25 @@
 public class ConfigExpanderTestCase {
 
     static ClassLoader classLoader;
-    static ConfigProviderResolver cpr;
-    Config config;
+    static ConfigProviderResolver configProviderResolver;
```

Comment:
I think we can drop all of the `Config` registration and just use `SmallRyeConfig` locally at each test method. If we look closely the `config` field doesn't seem to be used at all.

Reply:
Right, I was too focussed on the `catch`. 😄